python/cuml/naive_bayes/naive_bayes.py

```diff
@@ -334,6 +351,9 @@ def partial_fit(self,
                         Weights applied to individual samples (1. for
                         unweighted). Currently sample weight is ignored
 
+        convert_dtype : bool
+                        If True, convert y to the appropriate dtype (int)
+
```

Comment:
```suggestion
        convert_dtype : bool
            If True, convert y to the appropriate dtype (int)

```
I wouldnt indent this more than 4. There isnt much room already. Might want to fix all of them